### DISCUSSION TITLE
The credibility of the model

### DISCUSSION THREAD

**Discussion by lammps-vasp** [deepmodeling/deepmd-kit#659] [Q&A]
What is the upper limit of deepmd kit test error and training error, and how many orders of magnitude can be reached

**Comment by DingChangjie** [comment]
Hi, by "upper limit" do you mean how precise the DeepMD-kit could be ? Generally speaking, we care more on the error of force than that of energy. [Here is a video](https://www.bilibili.com/video/BV1CV411m7H7?from=search&seid=17767572293249764990), showing that the final error of force could reach an order of 10^(-3). However, you should also be careful of an overfitting problem, which appears to gain a small error but weaken the generalization ability remarkably ([See this issue](https://github.com/deepmodeling/dpgen/discussions/419))

**Reply by lammps-vasp** [reply]
Thank you very much

What is the maximum order of magnitude of the accuracy of deepmd kit?

In addition, I'm training with only energy information, no power information.

**Reply by DingChangjie** [reply]
You're welcome. Actually I'm also a learner in DeepMD, so some of my opinions may not always be correct...
1. Since DeepMD is based on a deep Artificial Neural Network, we quantify the "accuracy" by training error and testing error, which are reported by the program in the "lcurve.out"(column named l2_f_trn, l2_f_test, etc. the meanings are self-explanatory) . So _normally the smaller the better_, since large errors are skeptical, indicating a poor predicting ability. 
But this really depends on your own system. For my alloy system, the l2_f_test could reach a maximum order of 10^(-3) regardless of overfitting, which is the same as the video said.
2. By "power" do you mean the **virial** information ? If so, I recommend to try it, at least it really helped me  to decrease the training and testing error.
